docs/compatibility.md

```diff
@@ -21,6 +21,13 @@ You can find further information about released versions of ADF in the
 
 | ADF version | Content Services | Process Services |
 | --- | --- | --- |
+| [6.4.0](versionIndex.md#v640) | v7.4.0 | v2.3.8 |
+| [6.4.0](versionIndex.md#v640) | v7.3.0 | v2.3.5 |
+| [6.4.0](versionIndex.md#v640) | v7.2.0 | v2.2.0 |
+| [6.3.0] | v23.1.0 | v2.4.2 |
+| [6.0.0](versionIndex.md#v600) | v7.4.0 | v2.3.8 |
+| [5.1.0](versionIndex.md#v510) | v7.3.0 | v2.3.5 |
+| [5.0.0] | v7.3.0-M2 | v2.3.4 |
```

Comment:
@MateuszHY 
In the document, we have found ACS 7.3.0 M2 version and APS 2.3.4.
Based on that Test dashboard document we have performed the testing on mentioned versions
https://alfresco.atlassian.net/wiki/spaces/TECH/pages/1004109852/Goring+-+Test+Dashboard
We are checking the compatibility with ACS 7.2.0 and APS 2.3.4
Is this fine?

Reply:
I do not understand. @rbahirsheth 
My point is ... why did you use for testing 7.3.0-M2 version and nog 7.3.0? Did you test or assumed based on this doc you are linking that this is is compatible.

Reply:
Okay. I will use ACS 7.3.0 and perform testing.

Reply:
@MateuszHY My bad that we have used the 7.3.0-M2 version and due to multiple versions on this ticket it should have been 7.3.0 instead of M2. We created the compatibility matrix combination based on the released dashboards, for this case, it should have been 7.3.0.
We checked the compatibility matrix based on the created test suites.
Hope this answers your question.